hconfig_utils/mapl3g_hconfig_getter.F90

```diff
@@ -0,0 +1,235 @@
+#include "MAPL_ErrLog.h"
+module mapl3g_hconfig_getter
+
+   use :: pflogger, only: logger_t => logger
+   use :: esmf, MAXSTRLEN => ESMF_MAXSTR
+   use mapl_ErrorHandling
+
+   implicit none
+   public :: HConfigGetter
+   public :: get_value
+
+   type :: HConfigGetter
+      type(ESMF_HConfig) :: hconfig
+      character(len=:), allocatable :: label
+      logical :: found = .FALSE.
+      character(len=:), allocatable :: typestring
+      character(len=:), allocatable :: valuestring
+      logical :: value_equals_default = .FALSE.
+   end type HConfigGetter
+
+   interface get_value
+      module procedure :: get_value_i4
+   end interface get_value
+
+   character(len=*), parameter :: DEFAULT_FORMAT_STRING = '(G0)'
+   character(len=*), parameter :: EMPTY_STRING = ''
+
+   interface HConfigGetter
+      module procedure :: construct
+   end interface HConfigGetter
+
+contains
+   
+   type(HConfigGetter) function construct(hconfig, label, found)
+      type(ESMF_HConfig), intent(in) :: hconfig
+      character(len=*), intent(in) :: label
+      logical, intent(in) :: found
+
+      construct%hconfig=hconfig
+      construct%label=label
+      construct%found=found
+      construct%typestring=EMPTY_STRING
+      construct%valuestring=EMPTY_STRING
```

Comment:
please put space on either side of assignment `=`.  (But not when `=` is used for keywords in procedure references.)

Reply:
Done